folium/folium.py

```diff
@@ -465,15 +467,27 @@ def choropleth(self, geo_data, data=None, columns=None, key_on=None,
             Variable in the `geo_data` GeoJSON file to bind the data to. Must
             start with 'feature' and be in JavaScript objection notation.
             Ex: 'feature.id' or 'feature.properties.statename'.
-        threshold_scale: list, default None
-            Data range for D3 threshold scale. Defaults to the following range
-            of quantiles: [0, 0.5, 0.75, 0.85, 0.9], rounded to the nearest
-            order-of-magnitude integer. Ex: 270 rounds to 200, 5600 to 6000.
+        bins: int or sequence of scalars or str, optional
+            If `bins` is an int, it defines the number of equal-width
+            bins between the min and the max of the values (6, by default).
+            If `bins` is a sequence, it defines the bin edges,
+            including the rightmost edge, allowing for non-uniform bin widths.
+            For more information on this parameter, please have a look at
+            numpy.histogram function.
+            Warning: if any (non-nan) values is outside of the inclusive
```

Comment:
IMO you don't really have to state that here, they will see the error when it happens. Makes the docstring a bit less lenghty also.

Reply:
Better ?